mongomock/aggregate.py

```diff
@@ -952,7 +952,7 @@ def _handle_graph_lookup_stage(in_collection, database, options):
                 "Argument '%s' to $graphLookup must be string" % operator)
         if options[operator].startswith('$'):
             raise OperationFailure("FieldPath field names may not start with '$'")
-        if operator in ('connectFromField', 'as') and \
+        if operator in ('as') and \
```

Comment:
Thanks for implementing, please simplify the condition here (`==` instead of `in`) and make it fit on one line.

Note that `operator in ('as')` would return true only for operator being`a` or `s`, not `as` (that might be the missing line in coverage).